yoti-sdk-parent/pom.xml

```diff
@@ -222,8 +234,8 @@
           <artifactId>maven-compiler-plugin</artifactId>
           <version>${maven-compiler-plugin.version}</version>
           <configuration>
-            <source>${supported.java.version}</source>
-            <target>${supported.java.version}</target>
+            <source>${maven.compiler.source}</source>
+            <target>${maven.compiler.target}</target>
```

Comment:
as suggested there are 2 mvn profiles now 
- JDK8: sets the `source` and `target`
- JDK9+: sets the `release` so they are mutually exclusive

Additionally for the next major release the SDK version used to **build** should be raised as min to `11` or `17` even if we still need to keep the runtime to `8`. This is because the change in standards from JDK9, to drop the legacy properties and prepare the ground for a multi-version release (if needed)